src/init/CliRunner.ts

```diff
@@ -28,6 +28,26 @@ export class CliRunner {
     // Parse the command-line arguments
     const { argv: params } = yargs(argv.slice(2))
       .usage('node ./bin/server.js [args]')
+      .check((args, options): boolean => {
+        // Arguments that are not in the perdefined options are not allowed
+        if (args._ && args._.length > 0) {
+          throw new Error(`Arguments are not supported: "${args._.toString().split(',').join('", "')}"`);
+        }
+        for (const key in args) {
+          // _ and $0 are always present with yargs
```

Comment:
Yeah, but what are they 🙂 (i.e., I still need to look it up now)
Can we have something like
```suggestion
          // Skip filename arguments (_) and the script name ($0)
```